### PR TITLE
feat: added paragraph spacing support

### PR DESCRIPTION
**Pull request by L2jLiga**
*Description of changes:*

Created `paragraphStyle` as far as it same for `StringAttribute` and `TextStyle`. Added paragraph spacing property to it.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: models/ParagraphStyle/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+declare class ParagraphStyle {
+    _class: 'paragraphStyle';
+    alignment: number;
```
Comment (dbanksdesign) [inline models/ParagraphStyle/index.d.ts]: Should this use an enum like we do here: https://github.com/amzn/sketch-constructor/blob/master/models/BorderOptions/index.d.ts#L1

Comment (L2jLiga) [reply models/ParagraphStyle/index.d.ts]: yes, you're right :)

Comment (L2jLiga) [reply models/ParagraphStyle/index.d.ts]: fixed

**Pull request merged**